Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -602,29 +656,37 @@ void CGContextAddLines(CGContextRef context, const CGPoint* points, unsigned cou
  @Notes
 */
 CGPathRef CGContextCopyPath(CGContextRef context) {
-    NOISY_RETURN_IF_NULL(context, StubReturn());
-    UNIMPLEMENTED();
-    return StubReturn();
+    NOISY_RETURN_IF_NULL(context, nullptr);
+
+    if (!context->HasPath()) {
+        return nullptr;
+    }
+
+    return CGPathCreateCopy(context->Path());
 }
 
 /**
  @Status Stub
  @Notes
 */
 CGPoint CGContextGetPathCurrentPoint(CGContextRef context) {
-    NOISY_RETURN_IF_NULL(context, StubReturn());
-    UNIMPLEMENTED();
-    return StubReturn();
+    NOISY_RETURN_IF_NULL(context, CGPointZero);
+
+    if (!context->HasPath()) {
+        return CGPointZero;
+    }
```

Comment:
Yup, forgot about that.

---
In reply to: [85817685](https://github.com/Microsoft/WinObjC/pull/1240#discussion_r85817685) [](ancestors = 85817685)